### PR TITLE
RavenDB-21946 - Conflict resolution LastModifiedTime value is not updated after saving

### PR DESCRIPTION
**Pull request by shiranshalom**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-21946/Conflict-resolution-LastModifiedTime-value-is-not-updated-after-saving

### Type of change

- [x] Bug fix
- [ ] Regression bug fix
- [ ] Optimization
- [ ] New feature

### How risky is the change?

- [x] Low 
- [ ] Moderate 
- [ ] High
- [ ] Not relevant

### Backward compatibility

- [x] Non breaking change
- [ ] Ensured. Please explain how has it been implemented?
- [ ] Breaking change
- [ ] Not relevant

### Is it platform specific issue?

- [ ] Yes. Please list the affected platforms.
- [x] No

### Documentation update

- [ ] This change requires a documentation update. Please mark the issue on YouTrack using `Documentation Required` tag.
- [x] No documentation update is needed 

### Testing by Contributor

- [x] Tests have been added that prove the fix is effective or that the feature works
- [ ] Internal classes added to the test class (e.g. entity or index definition classes) have the lowest possible access modifier (preferable `private`) 
- [ ] It has been verified by manual testing

### Testing by RavenDB QA team

- [ ] This change requires a special QA testing due to possible performance or resources usage implications (CPU, memory, IO). Please mark the issue on YouTrack using `QA Required` tag.
- [x] No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- [ ] Yes. Please list the affected features/subsystems and provide appropriate explanation
- [x] No

### UI work

- [ ] It requires further work in the Studio. Please mark the issue on YouTrack using `Studio Required` tag.
- [x] No UI work is needed

### CODE REVIEW

**Review by garayx — APPROVED** [review]

File: src/Raven.Client/Documents/Operations/Configuration/GetConflictSolverConfigurationOperation.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Json.Serialization;
+using Raven.Client.ServerWide;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations.Configuration
+{
+    public sealed class GetConflictSolverConfigurationOperation : IMaintenanceOperation<ConflictSolver>
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Operations/Configuration/GetConflictSolverConfigurationOperation.cs]: move this to tests, at this point I do not see a reason to expose this

File: test/SlowTests/Issues/RavenDB_21946.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+                var conflictSolver = await store.Maintenance.SendAsync(new GetConflictSolverConfigurationOperation());
+
+                Assert.False(conflictSolver.IsEmpty());
+                Assert.False(conflictSolver.ResolveToLatest);
+                Assert.Single(conflictSolver.ResolveByCollection);
+                Assert.True(conflictSolver.ResolveByCollection.TryGetValue("Users", out var scriptResolver1));
+
+                // wait to confirm LastModifiedTime remains unchanged over time
+                await Task.Delay(1000);
```
Comment (ppekrol) [inline test/SlowTests/Issues/RavenDB_21946.cs:50]: use WaitForValueAsync instead

File: src/Raven.Client/Documents/Operations/Configuration/GetConflictSolverConfigurationOperation.cs (hunk 1 above)
Comment (shiranshalom) [reply src/Raven.Client/Documents/Operations/Configuration/GetConflictSolverConfigurationOperation.cs]: done :)

File: test/SlowTests/Issues/RavenDB_21946.cs (hunk 2 above)
Comment (shiranshalom) [reply test/SlowTests/Issues/RavenDB_21946.cs:50]: I believe using `WaitForValueAsync` might miss the intended purpose here.. the goal of this delay is to confirm that the time remains unchanged over a specific period, whereas `WaitForValueAsync` is designed to return the expected result as soon as it’s available

**Pull request merged**